chrome/content/zotero/xpcom/plugins.js

```diff
@@ -384,7 +397,64 @@ Zotero.Plugins = new function () {
 			: REASONS.ADDON_DOWNGRADE;
 	}
 	
-	
+
+	// TODO: Get blocking list from server
+	const BLOCKED_PLUGINS = {
+		"zoterostyle@polygon.org": {
+			versionRanges: [{
+				maxVersion: "4.5.99"
+			}],
+			reason: "This plugin, up to version 4.6.0, violates Zotero's plugin policies by severely break user interface.",
```

Comment:
I think we can just say `Versions of this plugin prior to version 4.6.0 break the Zotero user interface.`

Reply:
updated